core/src/main/java/nl/nn/adapterframework/senders/ShadowSender.java

```diff
@@ -200,20 +201,26 @@ private Message collectResults(Map<ISender, ParallelSenderExecutor> executorMap,
 	}
 
 	protected void addResult(SaxDocumentBuilder builder, ISender sender, Map<ISender, ParallelSenderExecutor> executorMap, String tagName) throws SAXException, IOException {
-		try (SaxElementBuilder senderResult = builder.startElement(tagName)) {
+		try (SaxElementBuilder resultXml = builder.startElement(tagName)) {
 			ParallelSenderExecutor pse = executorMap.get(sender);
 
 			StatisticsKeeper sk = getStatisticsKeeper(sender);
-			senderResult.addAttribute("duration", ""+sk.getLast());
+			resultXml.addAttribute("duration", ""+sk.getLast());
 
-			senderResult.addAttribute("senderClass", ClassUtils.nameOf(sender));
-			senderResult.addAttribute("senderName", sender.getName());
+			resultXml.addAttribute("senderClass", ClassUtils.nameOf(sender));
+			resultXml.addAttribute("senderName", sender.getName());
 			Throwable throwable = pse.getThrowable();
 			if (throwable==null) {
-				Message result = pse.getReply();
-				senderResult.addValue(XmlUtils.skipXmlDeclaration(result.asString()));
+				SenderResult senderResult = pse.getReply();
+				resultXml.addAttribute("success", Boolean.toString(senderResult.isSuccess()));
```

Comment:
kan je deze niet beter buiten de if zetten?

Reply:
Daar is hij niet nodig